docs/concepts/operator.md

```diff
@@ -161,9 +161,32 @@ This configuration allows for precise control over various extension parameters,
 
 The `.spec.deployment.admission` defines how an extension's admission controller may be deployed by the `gardener-operator`.
 The deployment of an admission controller is optional and may be omitted.
-Typically, the admission controllers run in the runtime cluster alongside the virtual cluster deployment.
-However, to function properly, the admission controllers require access and permissions for the virtual garden cluster in order to watch the Gardener API resources like `Shoot`s.
-As of today, deployment of admission controllers via `.spec.deployment.admission` is not yet supported, but it is currently under active development.
+Typically, the admission controllers are split in two parts:
+
+##### Runtime
+The `runtime` part contains deployment relevant manifests, required to run the admission service in the runtime cluster.
+The following values are passed to the chart during reconciliation:
+
+```yaml
+gardener:
+  runtimeCluster:
+    priorityClassName: <Class to be used for admission controller>
+```
+
+##### Virtual
+The `virtual` part includes the webhook registration ([MutatingWebhookConfiguration`/`Validatingwebhookconfiguration](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)) and RBAC configuration.
```

Comment:
I understand the motivation, but I also fear that it could make the whole thing more complex and less understandable (one might wonder where the permissions come from).